src/binary-encoding.adoc

```diff
@@ -887,24 +891,30 @@ consume only enough bits to represent the fractional precision supported by the
 30 (ns) +
 | Fractional second
 
-| `-`
+| `.`
 | n/a
 | Unused
 |===
 
-Timestamps are encoded in little-endian byte order. In the diagrams below, the first byte shows the leading opcode
-in hexadecimal; the trailing bytes show the meaning and layout of their respective bits using the letter codes defined
-in the table above.
+We will denote the timestamp encoding as follows with each byte ordered from vertically from top to bottom.  The
+respective bits are denoted using the letter codes defined in the table above.
 
-[source,%unbreakable]
+[%unbreakable,source]
 ----
-  byte 1           byte 2                         byte N
-+========+========================+=====+========================+
-| OPCODE | high nibble:low nibble | ... | high nibble:low nibble |
-+========+========================+=====+========================+
+          7       0 <--- bit position
+          |       |
+         +=========+
+byte 0   |  0xNN   | <-- hex notation for constants like opcodes
+         +=========+ <-- boundary between some encoding primitive (e.g., opcode/`FlexUInt`)
```

Comment:
> between some encoding primitive (...) 

The word "between" made me expect a second clause. ("Between some encoding primitive and ________").

Reply:
I will edit this as:

> boundary between encoding primitives (e.g., opcode/`FlexUInt`)